### PR TITLE
Bump adm-zip dependency version

### PR DESCRIPTION
**Pull request by leebickmtu**
Bump adm-zip dependency version to resolve `npm audit` vulnerability listing https://nodesecurity.io/advisories/681

### CODE REVIEW

**Comment by Sirikon** [comment]
Hey @leebickmtu sorry for the delay, thanks for reaching out 😄

Tested locally (`node test.js`) and it doesn't seem to work with `adm-zip 0.4.9`, but it works with `0.4.11`. Please, bump the version to `0.4.11` in your Pull Request and I'll accept it.

Best regards!

File: package.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   "description": "Node.js module for creating .zip files from a object tree",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "repository": {
     "url": "https://github.com/Sirikon/ziptree"
   },
   "main": "./lib/ziptree",
   "dependencies": {
-    "adm-zip": "0.4.4"
+    "adm-zip": "0.4.9"
```
Comment (Sirikon) [inline package.json]:
```suggestion
    "adm-zip": "0.4.11"
```

**Comment by leebickmtu** [comment]
Changed the adm-zip version to v0.4.11

**Comment by Sirikon** [comment]
@leebickmtu Re-bumped the ziptree version because had a little error while publishing the `0.1.1`.

`0.1.2` is published. Thanks!

**Pull request merged**